virtusize/src/main/java/com/virtusize/android/data/local/VirtusizeParams.kt

```diff
@@ -38,7 +38,6 @@ data class VirtusizeParams(
             "$PARAM_BID: '$bid', " +
             (if (sessionData != null) "$PARAM_SESSION_DATA: $sessionData, " else "") +
             "$PARAM_STORE_PRODUCT_ID: '${product.productCheckData?.productId}', " +
-            (if (externalUserId != null) "$PARAM_EXTERNAL_USER_ID: '$externalUserId', " else "") +
```

Comment:
Passing the user ID to the web view vsParamsFromSDK somehow caused https://app.clickup.com/t/3702259/NSDK-242
I don't know why. Removing this is judged by my observation.